source/sdk/structs/List.ooc

```diff
@@ -194,10 +194,12 @@ List: abstract class <T> extends BackIterable<T> {
 
         indexes := ArrayList<Int> new()
         for(i in 0..getSize()) indexes add(i)
+        currentIndex := Time microtime()
 
         while(!indexes empty?()) {
-            i := Random randRange(0, indexes getSize())
-            shuffled add(this[indexes removeAt(i)])
+            currentIndex = 214013 * currentIndex + 2531011 // Copied directly from source/math/IntRandomGenerator
+            currentIndex = ((currentIndex >> 16) & Short maximumValue) % indexes getSize()
+            shuffled add(this[indexes removeAt(currentIndex)])
```

Comment:
You could have simply removed the whole `shuffle` function and no one would have noticed, but alright, why not.